sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateUtils.scala

```diff
@@ -28,6 +28,7 @@ import org.apache.spark.sql.catalyst.expressions.Cast
  */
 object DateUtils {
```

Comment:
since we are supporting timestamp here, Maybe we want to rename this object to DateTimeUtils, and also update the javadoc.